Bugzilla/Config/Common.pm

```diff
@@ -151,21 +153,25 @@ sub check_opsys {
 
 sub check_bug_status {
   my $bug_status = shift;
-  my @closed_bug_statuses = map { $_->name } closed_bug_statuses();
-  if (!grep($_ eq $bug_status, @closed_bug_statuses)) {
-    return "Must be a valid closed status: one of "
-      . join(', ', @closed_bug_statuses);
-  }
+  try {
+    my @closed_bug_statuses = map { $_->name } closed_bug_statuses();
+    if (!grep($_ eq $bug_status, @closed_bug_statuses)) {
+      return "Must be a valid closed status: one of "
+        . join(', ', @closed_bug_statuses);
+    }
+  };
```

Comment:
Should we `catch` for a specific exception type? [It sounds like `try` without `catch` is supposed to be for exceptional scenarios](https://metacpan.org/pod/Try::Tiny):

> You can also use it like a standalone eval to catch and ignore any error conditions. Obviously, this is an extreme measure not to be undertaken lightly: